feature-definition-and-test-kit/testData/productcatalog-v1beta3-sman/Chart.yaml

```
@@ -0,0 +1,28 @@
+apiVersion: v2
+name: productcatalog
+description: A reference example TMFC001-ProductCatalogManagement ODA Component
```

To make the difference to the other productcatalog explicit I suggest to add at least a small comment:

```suggestion
description: A reference example TMFC001-ProductCatalogManagement ODA Component with secrets management
```